Adding starting and stopping database state (#680)
neo4j/docs-cypher: modules/ROOT/pages/administration/databases.adoc

| BOOLEAN

| requestedStatus

> | The expected status of the database. The value can be either `online` of `offline`. label:default-output[]

Reviewer: ```suggestion
| The expected status of the database. 
The value can be either `online` or `offline`. label:default-output[]
```

| STRING

| currentStatus
| The actual status of the database. label:default-output[]

The value can be one of the following:

* `online` - The database is running.
* `offline` - The database is not running.
* `starting` - The database is not running, but is about to be.

> * `stopping` - The database is not running anymore, but still not stopped completely.

Reviewer 2: Same here, I think you need a has between the but and still

No offline operations (e.g. `load`/`dump`) can be performed yet.

> * `store copying` - The database is currently being updated from another instance of Neo4j.

Reviewer 2: Do we have docs on store copying we could link here? Feel free to ignore this comment.

Author: Did not find anything, what I can link

> * `online` - The database is running.
> * `offline` - The database is not running.
> * `starting` - The database is not running, but is about to be.
> * `stopping` - The database is not running anymore, but still has not stopped completely.
> No offline operations (e.g. `load`/`dump`) can be performed yet.
> * `store copying` - The database is currently being updated from another instance of Neo4j.

Reviewer: ```suggestion
* `online` - the database is running
* `offline` - the database is not running
* `starting` - the database is not running, but is about to.
* `stopping` - the database is not running anymore, but still has not stopped completely.
Offline operations (e.g. `load`/`dump`) cannot be performed yet.
* `store copying` - the database is currently being updated from another instance of Neo4j
```

> * For other error states please refer to the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/errors/#database-management-states[operation manual].

Reviewer 2: I'd maybe drop the "error" here, I'm being pedantic but it sort of implies that the states above are also error states
